packages/@react-spectrum/badge/chromatic/Badge.chromatic.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Badge} from '../';
+import CheckmarkCircle from '@spectrum-icons/workflow/src/CheckmarkCircle';
+import {Flex} from '@react-spectrum/layout';
+import React from 'react';
+import {SpectrumBadgeProps} from '@react-types/badge';
+import {storiesOf} from '@storybook/react';
+import {Text} from '@react-spectrum/text';
+
+let variants = ['yellow', 'fuchsia', 'indigo', 'seafoam', 'magenta', 'purple', 'neutral', 'info', 'positive', 'negative'];
+
+storiesOf('Badge', module)
```

Comment:
please write in the CSF 3.0 format, see https://github.com/adobe/react-spectrum/pull/3129